contracts/base/MarketsState.sol

```diff
@@ -0,0 +1,136 @@
+pragma solidity 0.5.17;
+pragma experimental ABIEncoderV2;
+
+// Libraries
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/access/roles/WhitelistedRole.sol";
+
+// Contracts
+
+// Interfaces
+import "../interfaces/MarketsStateInterface.sol";
+import "../util/MarketStateLib.sol";
+
+
+/**
+    @notice This contract is used to store market data.
+
+    @author develop@teller.finance
+ */
+contract MarketsState is MarketsStateInterface, WhitelistedRole {
```

Comment:
When is WhiteListeRole set?

Reply:
This will be set in the deployment process.